src/components/Landing/styles.scss

```
@@ -0,0 +1,75 @@
+// Mobile portrait
+.wrapper {
+	display: flex;
+	width: 100%;
+	height: 100%;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+	background-color: #FF60D7;
+	font-size: 1.25rem;
+	line-height: 1;
+}
+
+	.heading, .subheading {
+		font-family: var(--font_family__heading);
+	}
+
+	.heading {
+		margin-bottom: 0.1em;
+		font-size: 2.9em;
+		letter-spacing: 0.1em;
+		text-transform: uppercase;
+	}
+
+	.subheading {
+		margin-bottom: 0.5em;
```

```suggestion
		margin-bottom: 1em;
```
I'm getting `45px` here, so a nice, round `1em` from me.